users/config.rst

```diff
@@ -408,6 +409,7 @@ Device Element
         <maxSendKbps>0</maxSendKbps>
         <maxRecvKbps>0</maxRecvKbps>
         <maxRequestKiB>0</maxRequestKiB>
+        <remoteGUIPort>8384</remoteGUIPort>
```

Comment:
I would change this to `<remoteGUIPort>0</remoteGUIPort>`, but also add `<remoteGUIPort>8384</remoteGUIPort>` to the next device config below. This way we have both the default and a custom values shown.